tcf_website/static/css/site/tokens.css

```
@@ -0,0 +1,265 @@
+/*
+ * theCourseForum Design Tokens
+ * ============================
+ * Central source of truth for all design values.
+ * Import this file first in any stylesheet.
+ */
+
+:root {
+  /* ===== COLOR TOKENS ===== */
+  
+  /* Base palette */
+  --color-white: #ffffff;
```

Stylelint (17.2.0)

[error] Expected "#ffffff" to be "#fff" (color-hex-length)

(color-hex-length)